content/scalability/docs/supporting-infrastructure.md

```diff
@@ -0,0 +1,228 @@
+# Supporting Infrastructure
+
+Supporting infrastructure is resources required for your workloads to run and be accessible. It includes EC2 instances, load balancers, external storage, and other APIs used by the Kubernetes control plane. For simplicity, we won’t address APIs that workloads use because that could be every API.
+
+Helping EKS clusters scale to large sizes requires using the supporting infrastructure wisely. There are multiple areas you need to consider in planning before they become a problem. The main concerns are for compute availability and network routing.
+
+## EC2 instances
+
+Selecting your EC2 instance types is possibly one of the hardest decisions customers face because in clusters with multiple different workloads there is no one-size-fits all solution. Here are some tips to help you avoid common pitfalls.
+
+### Use Karpenter for node autoscaling
+
+[Karpenter](https://karpenter.sh/) is a workload-native node autoscaler created by AWS. It can scale nodes in a cluster based on the workload requirements for resources (e.g. GPU) and taints and tolerations (e.g. zone spread) without needing to manage node groups for each workload requirement. Nodes are created directly from EC2 instead of using Auto Scaling Groups (ASG) which avoids default node group quotas—450 nodes per group—and provides greater instance selection flexibility with less operational overhead.
```

Comment:
We don't want to be too critical of MNGs. They're still way more popular that Karpenter.

Reply:
Do you feel like this is being critical? I was trying to state a limit for large clusters.

Reply:
it feels heavily biased toward Karpenter, which is fine - we want customers to use Karpenter. At the same time, we don't want customers who have chosen MNGs to question their decision. Let's try to make it more balanced if possible.  @rothgar